extensions/iceberg/src/main/java/io/deephaven/iceberg/location/IcebergTableParquetLocationKey.java

```diff
@@ -18,24 +20,50 @@
 public class IcebergTableParquetLocationKey extends ParquetTableLocationKey implements IcebergTableLocationKey {
     private static final String IMPLEMENTATION_NAME = IcebergTableParquetLocationKey.class.getSimpleName();
 
+    /**
+     * Following are derived from the {@link DataFile} that backs the keyed location and are used for ordering of data
```

Comment:
I would use a regular comment, or document all the fields individually to explain what they are (might be better).